### PR TITLE
Architecture/Deployments: describe the way deployments service is selecting deployment for the device

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Issue: https://tracker.mender.io/browse/MEN-3327

### CODE REVIEW

File: 02.Architecture/05.Deployments/docs.md [hunk 1]
```diff
@@ -84,4 +84,12 @@ The following values for the status field for a finished update are:
 * "noartifact"
 * "already-installed"
 * "aborted"
-* "decommissioned"
\ No newline at end of file
+* "decommissioned"
+
+### Alghoritm for selecting the Deployment for the Device
```
Comment (drewmoseley) [inline 02.Architecture/05.Deployments/docs.md:89]: Algorithm

File: 02.Architecture/05.Deployments/docs.md [hunk 2]
```diff
@@ -84,4 +84,12 @@ The following values for the status field for a finished update are:
...
+
+It is possible that the device has been targeted for one deployment or more.
+When the device is asking for the deployment, deployments service looks for the oldest, not finished deployment the device was targeted for.
```
Comment (drewmoseley) [inline 02.Architecture/05.Deployments/docs.md:92]: the deployments service looks for the oldest

File: 02.Architecture/05.Deployments/docs.md [hunk 3]
```diff
@@ -84,4 +84,12 @@ The following values for the status field for a finished update are:
...
+If there is one, the deployments service creates the instance of the deployment for the device (device deployment) with a "pending" state and executes following operations:
```
Comment (drewmoseley) [inline 02.Architecture/05.Deployments/docs.md:93]: the following operations:

File: 02.Architecture/05.Deployments/docs.md (hunk 1 above)
Comment (kjaskiewiczz) [reply 02.Architecture/05.Deployments/docs.md]: fixed

File: 02.Architecture/05.Deployments/docs.md (hunk 2 above)
Comment (kjaskiewiczz) [reply 02.Architecture/05.Deployments/docs.md]: fixed

File: 02.Architecture/05.Deployments/docs.md (hunk 3 above)
Comment (kjaskiewiczz) [reply 02.Architecture/05.Deployments/docs.md]: fixed

**Comment by drewmoseley** [comment]
LGTM. :+1:

**Comment by drewmoseley** [comment]
LGTM.

**Comment by kjaskiewiczz** [comment]
@drewmoseley github still shows that you requested changes

**Comment by drewmoseley** [comment]
@kjaskiewiczz I found the button to click to turn that off.

**Comment by kjaskiewiczz** [comment]
@drewmoseley thanks for the review!

**Pull request merged**